### DISCUSSION TITLE
Why isn't the diagram shown on Dropbox shared HTML file

### DISCUSSION THREAD

**Discussion by danddinh** [jgraph/drawio#3881] [Q&A] [answered]
Dropbox allows people to share HTML files besides pdf, doc, sheets, etc.

Draw.io embed works perfectly when it is inside a regular HTML file served by a static server, however, it is not shown on Dropbox HTML file. How to make it work with Dropbox, I wish to share HTML files to friends.

Example URL: https://www.dropbox.com/scl/fi/coy2uje9z55kstzmas7n5/Test-Nofe-1.html?rlkey=vp8t1fbgo1zjcr0so3xa8ipuw&dl=0

The issue is shown as:
![image](https://github.com/jgraph/drawio/assets/116571017/2d82be4f-9305-4209-89d7-64186e45d79c)

**Comment by davidjgraph** [comment, marked as answer]
This is a custom viewer specific to dropbox, it's either not executing Javascipt (most likely) or has limited JS support. You'd be better storing the diagram as PNG+XML or SVG+XML.

**Reply by danddinh** [reply]
Oh I got it, no JS
